nautobot/dcim/models/locations.py

```diff
@@ -214,6 +216,34 @@ class Location(TreeNode, StatusModel, PrimaryModel):
         null=True,
     )
     description = models.CharField(max_length=200, blank=True)
+    facility = models.CharField(max_length=50, blank=True, help_text="Local facility ID or description")
+    asn = ASNField(
+        blank=True,
+        null=True,
+        verbose_name="ASN",
+        help_text="32-bit autonomous system number",
+    )
+    time_zone = TimeZoneField(blank=True)
+    physical_address = models.CharField(max_length=200, blank=True)
+    shipping_address = models.CharField(max_length=200, blank=True)
```

Comment:
Let's consider whether this should instead be a `TextField`? 200 chars isn't much for some addresses.